modern-cpp-font-lock.el

```diff
@@ -53,85 +53,104 @@
   "Provides font-locking as a Minor Mode for Modern C++"
   :group 'faces)
 
-(defun modern-c++-string-lenght< (a b) (< (length a) (length b)))
-(defun modern-c++-string-lenght> (a b) (not (modern-c++-string-lenght< a b)))
-
-(defcustom modern-c++-types
-  (eval-when-compile (sort '("bool" "char" "char16_t" "char32_t" "double" "float" "int" "long" "short" "signed" "unsigned" "void" "wchar_t")
-                           'modern-c++-string-lenght>))
+(eval-and-compile
```

Comment:
This two functions need at both compile time and runtime.
`eval-and-compile` do the job.